docs/user/openstack/README.md

```diff
@@ -69,6 +69,27 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
 
 ## OpenStack Requirements
 
+The OpenShift installation on OpenStack platform relies on a number of core
+services being available:
+
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+  - Placement, as it is required by Nova
```

Comment:
Honestly, I would just remove Placement as we're not directly making calls to this service (it just happens to be a hard requirement for Nova).